test/msgpuck.h

```diff
@@ -0,0 +1,3193 @@
+#ifndef MSGPUCK_H_INCLUDED
+#define MSGPUCK_H_INCLUDED
```

Comment:
Let's use CMake `FetchContent` to download `msgpuck` during test build.

Reply:
Agreed, but please let's do it separately. Please file an isssue.